tests/integration/security/https_jwt/main_test.go

```diff
@@ -46,6 +46,7 @@ func loadCert(filename string) (string, error) {
 func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
+		Skip("https://github.com/istio/istio/issues/36829").
```

Comment:
Is the test still flaky? it seems it only failed once in the past week: https://k8s-testgrid.appspot.com/istio_istio_postsubmit#integ-k8s-123_istio_postsubmit&width=20&include-filter-by-regex=TestJWTHTTPS

Also I think @aryan16 has a PR to improve the test by adding the readiness probe